NAMESPACE

```diff
@@ -121,22 +117,33 @@ import(circlize)
 import(config)
 import(cowplot)
 import(data.table)
-import(dbplyr)
 import(digest)
 import(dplyr)
+import(forcats)
 import(g3viz)
 import(ggbeeswarm)
 import(ggplot2)
 import(ggpubr)
 import(ggrepel)
 import(ggsci)
 import(ggthemes)
+import(glue)
 import(grid)
+import(gridExtra)
 import(maftools)
 import(metaviz)
 import(parallel)
 import(plotly)
+import(purrr)
+import(readr)
+import(reshape2)
 import(rtracklayer)
-import(tidyverse)
+import(ssh)
```

Comment:
I might misremember this - but I think there was some issue with installing the `ssh` package on GSC. So we opted to not import it and leave it to remote users to take care of. Maybe I am not correct or this has been resolved - but I would keep this out of the dependency list

Reply:
Right, thanks for pointing this out. I will make the changes necessary